manifest/989onan/NoKnockBack/info.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "989onan.NoKnockBack",
+	"id": "989onan.NoKnockBack",
+	"description": "A Resonite mod that prevents knockback and sends an impulse of \"knockbackcanceled\" for knockback and \"gravitychangecanceled\" for gravity changes to the client's user root.",
+	"category": "Misc",
+	"sourceLocation": "https://github.com/989onan/NoKnockBack",
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/989onan/NoKnockBack/releases/tag/1.0.0",
+			"artifacts": [{
+				"url": "https://github.com/989onan/NoKnockBack/releases/download/1.0.0/NoKnockBack.dll",
+				"sha256": "0857A5B614F221CFC9B77B5896118691F9659E2E54AF59857A3DD477B12EC399"
+			}]
+		},
+        "1.1.0": {
```

Comment:
File info set incorrectly
Built in debug mode
Version does not match version being added
Several Msg logging calls that may be left over from testing,